addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java

```diff
@@ -928,15 +923,15 @@ public static void extractJsonTokens(
 
     private static void extractJsonTokens(
             Object obj, String parent, Map<String, SessionToken> tokens) {
-        if (obj instanceof JSONObject) {
-            extractJsonTokens((JSONObject) obj, parent, tokens);
-        } else if (obj instanceof JSONArray) {
-            Object[] oa = ((JSONArray) obj).toArray();
+        if (obj instanceof JSONObject jsonObj) {
```

Comment:
Would be good if there were some consistency in the naming.

Reply:
I had thought I had tried to be but I'll go through them again

Reply:
Some of them were using j prefix.